### PR TITLE
creating docs for usage API

### PR DESCRIPTION
**Pull request by victorchaplin**

### CODE REVIEW

File: spec/components/parameters/endDate.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import { ParameterObject } from 'openapi3-ts';
+import { createComponentRef } from '../../../utils/ref';
+
+const endDate: ParameterObject = {
+  name: 'endDate',
```
Comment (ghilhotina) [inline spec/components/parameters/endDate.ts]: In order to respect the pattern used in the previous APIs, in my opinion, would be better to call this parameter 'endAt'.

File: spec/components/parameters/startDate.ts [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import { ParameterObject } from 'openapi3-ts';
+import { createComponentRef } from '../../../utils/ref';
+
+const startDate: ParameterObject = {
+  name: 'startDate',
```
Comment (ghilhotina) [inline spec/components/parameters/startDate.ts]: In order to respect the pattern used in the previous APIs, in my opinion, would be better to call this parameter as 'startAt'.

File: spec/tags/groups.ts [hunk 3]
```diff
@@ -12,6 +12,11 @@ const groups: TagGroupObject[] = [{
   tags: [
     'Subscriptions',
   ],
+}, {
+  name: 'Reports',
+  tags: [
+    'Usage',
```
Comment (rafael-org) [inline spec/tags/groups.ts]: Faltou atualizar a tag eu diria, não?

File: spec/paths/reports/reports@usage.ts [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+import { ref as groupByRef } from '../../components/parameters/groupBy';
+import { ref as ticketTypesRef } from '../../components/parameters/ticketTypes';
+import { ref as channelsRef } from '../../components/parameters/channels';
+import { ref as usageResponseRef } from '../../components/schemas/reports/usage';
+
+const get: OperationObject = {
+  summary: 'Get usage report',
+  description:
+    'Currently, only usage for Web Chat, SMS, WhatsApp and Facebook is available. More channels will be added later. Error messages will provide details about any problems with the request.',
+  tags: ['Usage'],
```
Comment (rafael-org) [inline spec/paths/reports/reports@usage.ts]: Aqui também acho que deveria ser 'Reports' também né? Igual no spec/tags/index.ts

**Comment by rafael-org** [comment]
Closing old PRs...

**Pull request closed without merging**